edbdeploy/project.py

```diff
@@ -409,6 +470,45 @@ def _build_terraform_vars(self, env):
                 aws_region=env.aws_region,
                 aws_ami_id=getattr(env, 'aws_ami_id', None) or None,
             ))
+        # AWS RDS case
+        if env.cloud == 'aws-rds':
+            self.terraform_vars.update(dict(
+                aws_image=os_spec['image'],
+                aws_region=env.aws_region,
+                aws_ami_id=getattr(env, 'aws_ami_id', None) or None,
+            ))
+
+            if env.shirt == 'small':
+                self.terraform_vars.update(dict(
+                    guc_effective_cache_size='524288',
+                    guc_shared_buffers='3145728',
+                    guc_max_wal_size='51200',
+                ))
+            elif env.shirt == 'medium':
+                self.terraform_vars.update(dict(
+                    guc_effective_cache_size='4718592',
+                    guc_shared_buffers='3145728',
+                    guc_max_wal_size='102400',
+                ))
+            elif env.shirt == 'large':
+                self.terraform_vars.update(dict(
+                    guc_effective_cache_size='13107200',
+                    guc_shared_buffers='3145728',
+                    guc_max_wal_size='204800',
+                ))
+            elif env.shirt == 'xl':
+                self.terraform_vars.update(dict(
+                    guc_effective_cache_size='29884416',
+                    guc_shared_buffers='3145728',
+                    guc_max_wal_size='409600',
+                ))
```

Comment:
Does this part only apply to RDS and not to Aurora as well?

It would be good if you can move the hard coded values into `edbdeploy/spec/aws_rds.py` (and `edbdeploy/spec/aws_rds_aurora.py` ?).

```python
# edbdeploy/spec/aws_rds.py

# Sorry for this dirty variable name
RDS_HammerDB_TShirt_GUC = {
    'small': {
        'effective_cache_size': '524288',
        'shared_buffers': '3145728',
        'max_wal_size': '51200'
    },
    'medium': {
      ...
   },
  ...
}
```

```python
# edbdeploy/project.py
from .spec.aws_rds import RDS_HammerDB_TShirt_GUC

...

self.terraform_vars.update(dict(
    guc_effective_cache_size=RDS_HammerDB_TShirt_GUC[env.shirt]['effective_cache_size'],
    guc_shared_buffers=RDS_HammerDB_TShirt_GUC[env.shirt]['shared_buffers'],
    guc_max_wal_size_size=RDS_HammerDB_TShirt_GUC[env.shirt]['max_wal_size'],
))
```

Reply:
Updated in 05610127a80be3c1a122a2de38c4d4dc68b66f67